### PR TITLE
first app 2

### PR DESCRIPTION
**Pull request by statu**
Found out tonight that my stuff didn't save :(  Had to start over but need to go to bed.

### CODE REVIEW

**Comment by statu** [comment]
Please give me feedback. I was still lost here.

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+(function () {
+    
+       var data = null;
+
+    var object = kevinSmithMovies.items;
+    var localStorageKey =
```
Comment (mbMosman) [inline js/script.js]: Missing a value here causing a syntax error, so the following code doesn't run.

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    
+    
+    var headingText = document.createTextNode(title);
+    var h3.appendChild(headingText);
+    var target.appendChild(h3);
+    var ul = document.createElement("ul");
+    var stats = document.createTextNode("Director: " + director + ", "  + "Producer: " + producer + ", " + "Released: " + year + ", " + "Cast: " + cast);
+        ul.appendChild(stats);
+        target.appendChild(ul);
+        }
```
Comment (mbMosman) [inline js/script.js]: I think these are misplaced }'s

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+
+(function () {
```
Comment (mbMosman) [inline js/script.js]: You needed to build an application object for your data and methods, similar to the one illustrated in the book for the address book application.

**Comment by mbMosman** [comment]
It doesn't run due to missing code?  syntax errors.

It looks amazingly similar to my simple ajax app code...  It's fine to use an example, but it's also important to be able to understand how it works to modify it and combine multiple examples (such as mine & the one from the text) together.

**Pull request closed without merging**